crates/store/src/iter.rs

```diff
@@ -58,7 +58,10 @@ impl<'a, K, V> Iter<'a, K, V> {
 }
 
 // TODO: We should consider using std::iter::Iterator here
-#[allow(clippy::should_implement_trait)]
+#[expect(
+    clippy::should_implement_trait,
+    reason = "TODO: This should be implemented"
+)]
```

Comment:
uhm, why? we already have `.keys()` and `.entries()` that are iterators and represent the functionality we want

Reply:
Because of expectations is all... 🙂